add-product/main.c

```diff
@@ -1,8 +1,25 @@
 #include <stdio.h>
+#include <stdlib.h>
```

Comment:
What order we want to maintain in imports? (alphabet?)

Reply:
alphabetic libraries, then alphabetic local